### PR TITLE
Private data repo

### PR DESCRIPTION
**Pull request by trey-stafford**

### CODE REVIEW

File: CHANGELOG.md
Comment (MattF-NSIDC) [inline CHANGELOG.md:4]: Maybe add parenthetical:

```
 - Support adding layers from data contained in a private repository (just a special directory exposed as a docker volume mount for now).
```

"repository" comes with context that may confuse people initially.

File: qgreenland/tasks/layers/__init__.py
Comment (MattF-NSIDC) [inline qgreenland/tasks/layers/__init__.py:9]: If our code were better, we wouldn't have had to make a new pipeline for this. Continuing a pattern we already have, we could expose fetch tasks the same way we expose `INGEST_TASKS` here, and then we could mix-and-match fetch and ingest. And eventually, ingest can just be command pipelines! Maybe with exceptions. But maybe the baby step towards that baby step would be to expose fetch tasks at the configuration level. That would maybe eventually lead to us being able to get rid of the `file_type` config key? Or expose it as an `ingest_kwarg` so it's more clear what the heck it means.

File: scripts/seed_private_archive_from_prod.sh
Comment (MattF-NSIDC) [inline scripts/seed_private_archive_from_prod.sh:6]: Why this one have a trailing slash and before one not have?

Comment (MattF-NSIDC) [reply scripts/seed_private_archive_from_prod.sh:6]: If the trailing slash is a requirement of the `rsync` command, I suggest moving the trailing slash to the command so the variables are consistent. The trailing slash right now is a hidden requirement that someone else might see as an error.

File: qgreenland/config/datasets.yml
Comment (MattF-NSIDC) [inline qgreenland/config/datasets.yml:1039]: We should resolve this TODO before merging; dataset IDs end up in the input cache and are rarely cleaned up. Speaking of which, we should probably eventually write a cleanup tool to remove any dataset IDs in the input cache that aren't in the config.

File: qgreenland/config/layers.yml
Comment (MattF-NSIDC) [inline qgreenland/config/layers.yml:271]: I feel like we already have an empty group somewhere for this... `Environmental management/Exclusive economic zones`!

Comment (MattF-NSIDC) [reply qgreenland/config/layers.yml:271]: There's only one official "Exclusive Economic Zone" right? To everyone, that means this one thing? In that case, let's get rid of the group, and replace it with this layer in `Environmental management` group. If not, maybe we should further qualify the `id` and `title` of this one to `1973_agreement_exclusive_economic_zone`, and insert it into that group.

File: qgreenland/tasks/layers/__init__.py
Comment (trey-stafford) [reply qgreenland/tasks/layers/__init__.py:9]: Creating a task for this.

File: scripts/seed_private_archive_from_prod.sh
Comment (trey-stafford) [reply scripts/seed_private_archive_from_prod.sh:6]: Actually, this was just a mistake. I'll add the trailing slash to the first.

File: qgreenland/config/datasets.yml
Comment (trey-stafford) [reply qgreenland/config/datasets.yml:1039]: That's a good idea. We're waiting for feedback from Karl at the moment.

File: qgreenland/config/layers.yml
Comment (trey-stafford) [reply qgreenland/config/layers.yml:271]: Hmm, yeah. We'll need more info on this. Maybe the empty group was wrong, as this layer is in 'Misc' in our layer planning spreadsheet.

**Review by MattF-NSIDC — APPROVED** [review]

File: qgreenland/config/datasets.yml
Comment (trey-stafford) [reply qgreenland/config/datasets.yml:1039]: Created a task to make a script that will prune outdated datasets from the cache dir. This might be a good test-case for that, so I'm going to go ahead and merge this!

**Pull request merged**
